services/cd-service/pkg/repository/repository.go

```diff
@@ -894,6 +894,34 @@ func (s *State) GetEnvironmentApplicationLocks(environment, application string)
 	}
 }
 
+func (s *State) GetDeploymentMetaData(environment, application string) (string, time.Time, error) {
+	base := s.Filesystem.Join("environments", environment, "applications", application)
+	author, err := readFile(s.Filesystem, s.Filesystem.Join(base, "deployed_by"))
```

Comment:
No email here?

Reply:
Was not mentioned in the meeting we had on Thursday but can add now